cmds.py

```diff
@@ -43,64 +42,48 @@ def find_command(**kwargs):
         'reset': room.reset_chat_user,
         'mute': moderation.mute,
         'unmute': moderation.unmute,
+        'report_list': moderation.list_reports,
     }
     basic_commands = {
-        'help': other.help,
+        'help': other.help_command,
         'song': other.song,
         # 'refresh': online.refresh_online,
         # 'offline': online.appear_offline,
```

Comment:
I see the comment in `online_commands.py` now, so this is a reminder to change it here as well I guess

Reply:
I got them working. I had a issue where the user would not be set offline when i ran the command, but it ends up the issue was bigger than I thought. It was a easy fix but it was an issue that would cause pain in the future.